package.json

```diff
@@ -20,7 +20,7 @@
     "pg": "^7.12.0",
     "proxyquire": "^2.1.1",
     "sequelize": "^6.0.0",
-    "sinon": "^7.4.0"
+    "sinon": "^7.3.1"

```

Comment:
Why 7.3.1?  7.3.2 seems to be the latest stable. 
https://www.npmjs.com/package/sinon

Reply:
Oh yes, although it was 7.4.0 earlier! I'll get it changed.